.github/workflows/publish-qemu-bin.yml

```diff
@@ -0,0 +1,163 @@
+      uses: actions/upload-artifact@v4
+      with:
+        name: qemu-windows
+        path: |
+          temp\*
+        if-no-files-found: error
+
+  qemu-linux:
+    name: Build Qemu for Linux
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+    - name: Get current Qemu version
+      id: qemu-version
+      run: |
+        qemuVersion=$(jq -r .linux $QEMU_VERSION_FILE)
+        echo "QEMU_VERSION=$qemuVersion" >> $GITHUB_OUTPUT
+      shell: bash
+    - name: Install Dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install --yes --no-install-recommends \
+          autoconf \
+          automake \
+          autotools-dev \
+          bc \
+          build-essential \
+          dosfstools \
+          gcc \
+          libglib2.0-dev \
+          libgtk-3-dev \
+          libpixman-1-dev \
+          libsdl2-dev \
+          mtools \
+          ninja-build \
+          tar
+    - name: Compile Qemu ${{ steps.qemu-version.outputs.QEMU_VERSION }}
+      env:
+        QEMU_VERSION: ${{ steps.qemu-version.outputs.QEMU_VERSION }}
+        QEMU_URL: "https://gitlab.com/qemu-project/qemu.git"
+        DEBIAN_FRONTEND: "noninteractive"
+      run: |
+        git clone $QEMU_URL --branch $QEMU_VERSION --depth 1 qemu
+        cd qemu
+        mkdir bin
+        cd bin
+        ./../configure --target-list=x86_64-softmmu,aarch64-softmmu --enable-gtk
+        sudo make -j $(nproc)
+    - name: Stage Qemu Binaries
+      run: |
+        mkdir -p temp/share
+        cp qemu/bin/qemu-system-x86_64 qemu/bin/qemu-system-aarch64 temp
+        cp qemu/bin/qemu-bundle/usr/local/share/qemu/vgabios-cirrus.bin temp/share
+        cp qemu/bin/qemu-bundle/usr/local/share/qemu/vgabios-stdvga.bin temp/share
+        cp qemu/bin/qemu-bundle/usr/local/share/qemu/kvmvapic.bin temp/share
+    - name: Upload Qemu Artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: qemu-linux
+        path: |
+          temp/*
+        if-no-files-found: error
+  
+  prepare-extdep:
+    name: Prepare and Upload Qemu External Dependency
+    needs: [qemu-windows, qemu-linux]
+    runs-on: Ubuntu-latest
+    permissions:
+      contents: write
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install dependencies
+        run: |
+          pip install edk2-pytool-extensions
+      - name: Download Windows Qemu Artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: qemu-windows
+          path: windows
+      - name: Download Linux Qemu Artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: qemu-linux
+          path: linux
+      - name: Stage Ext-Dep
+        run: |
+          mkdir -p qemu
+          mv windows/ qemu/Windows/
+          mv linux/ qemu/Linux/
+          ls qemu/Windows
+          ls qemu/Linux
+          chmod a+x qemu/Linux/*
+      - name: Package Ext-Dep

```

Comment:
I think it may make sense to just have the linux dir in .tar.gz and just the windows dir in the .zip. Not point downloading both in any given ext-dep, and it would cut the download in half.

Reply:
I've split them into separate external dependencies:

`qemu-windows-$(release-version).zip`
`qemu-linux-$(release-version).tar.gz`